src/commands/statusline.ts

```
@@ -57,19 +56,13 @@ export const statuslineCommand = define({
 		}
 
 		// Extract session ID from transcript path
-		// Path format: /path/to/projects/{project}/{session}/transcript.json
-		const transcriptParts = hookData.transcript_path.split(path.sep);
-		const sessionId = transcriptParts[transcriptParts.length - 2] ?? '';
+		const sessionId = hookData.session_id;
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)